shard.go

```diff
@@ -117,6 +117,18 @@ func (s *cacheShard) getValidWrapEntry(key string, hashedKey uint64) ([]byte, er
 	return wrappedEntry, nil
 }
 
+func (s *cacheShard) contains(key string, hashedKey uint64) bool {
+	s.lock.RLock()
+	wrappedEntry, err := s.getWrappedEntry(hashedKey)
+	if err != nil {
+		s.lock.RUnlock()
+		return false
+	}
+	entryKey := readKeyFromEntry(wrappedEntry)
+	s.lock.RUnlock()
```

Comment:
@cristaloleg do you think we should increment hit count here?

Reply:
Deeply philosophical question. I'm fine to not increment it here. Leave it only for real value get.